examples/aditof-demo/aditofdemoview.h

```diff
@@ -91,6 +89,12 @@ class AdiTofDemoView {
     bool localModeChecked = false;
     bool ethModeChecked = false;
     int connectionCurrentValue = 4;
+
+    bool depthIrChecked = true;
+    bool depthOnlyChecked = false;
+    bool irOnlyChecked = false;
+    int frameTypeCurrentValue = 4;
+    // 4 = depthIr(default); 2 = depthOnly; 1 = irOnly
```

Comment:
can you put this comment on the same line as the line code?

Reply:
yes, that might be a little better

Reply:
fixed